ce/tests/test_api.py

```diff
@@ -119,6 +140,7 @@ def test_multimeta(populateddb, model):
     # make sure start_date and end_date are present
     assert 'start_date' in rv[unique_id]
     assert 'end_date' in rv[unique_id]
+    assert 'modtime' in rv[unique_id]
```

Comment:
Should you be asserting the types (datetime) of these?

Reply:
Yes, good suggestion.